src/core/display/Container.js

```diff
@@ -544,3 +544,12 @@ Container.prototype.renderCanvas = function (renderer)
         renderer.maskManager.popMask(renderer);
     }
 };
+
+/**
+ * Destroys the container
+ *
+ */
+Container.prototype.destroy = function ()
+{
+    this.children = null;
```

Comment:
I might be missing an obvious case here, but I thought I ask anyway ;)

Could be be recursive, or maybe recursive with an arg `Container.prototype.destroy = function (recursive)`, or maybe another method?

At the very least, should this not call `removeChildren`?